t2.lic

```diff
@@ -17,11 +17,9 @@ class T2
 
     @counter = 0
     @settings = get_settings
```

Comment:
Why not call `reload_settings` here so we check for validity at first startup (and not just on reload)?

Reply:
Probably not a bad idea, I've accidently run it before on characters that weren't setup for T2 yet and it took me a bit to figure out why it kept stowing things over and over.